tests/cbor_decode/test1_suit_old_formats/CMakeLists.txt

```diff
@@ -9,25 +9,46 @@ cmake_minimum_required(VERSION 3.13.1)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(NONE)
 
-set(BUILD_WITH_ZEPHYR ON)
-
-add_subdirectory(../../.. ${PROJECT_BINARY_DIR}/cddl_gen)
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE
+  ${app_sources})
 
-target_cddl_source(app ../../cases/manifest-moran3.cddl
-  ENTRY_TYPES OuterWrapper
-  DECODE
-  ${VERBOSE}
-  ${CANONICAL}
+set(py_command3
+  ${PYTHON_EXECUTABLE}
+  ${CMAKE_CURRENT_LIST_DIR}/../../../cddl_gen/cddl_gen.py
+  -c ${CMAKE_CURRENT_LIST_DIR}/../../cases/manifest-moran3.cddl
+  --default-max-qty 6
+  code
+  --output-cmake ${PROJECT_BINARY_DIR}/manifest-moran3.cmake
+  -t OuterWrapper
+  --decode
   )
-target_cddl_source(app ../../cases/manifest-moran4.cddl
-  ENTRY_TYPES SUIT_Outer_Wrapper
-  DECODE
-  ${VERBOSE}
-  ${CANONICAL}
+
+set(py_command4
+  ${PYTHON_EXECUTABLE}
+  ${CMAKE_CURRENT_LIST_DIR}/../../../cddl_gen/cddl_gen.py
```

Comment:
Can't this be replaced by the `cddl-gen` command, like in `test2_suit/CMakeLists.txt`?